Add a script to check a specific eslint rule

mwiencek opened the pull request:
This can be invoked like

```sh
./script/check_eslint_rule \
  'block-scoped-var: [warn]' \
  root/
```

to check a specific rule under a specific directory

yvanzo commented:
Using NodeJS v10.10.0:

```console
$ ./node_modules/.bin/eslint root/types.js
Error: /home/yvan/musicbrainz-server/.eslintrc.yaml:
        Configuration for rule "spaced-comment" is invalid:
        Value {"null":null} should be string.
```
It might be a regression caused by merged commit https://github.com/metabrainz/musicbrainz-server/pull/1244/commits/1cc0bb28988ec0a70f9c1fcc7b02bd6f65774d10.

reosarevok commented:
I have a change for that configuration on the spaces PR, not sure whether that'd change anything though :)

mwiencek commented:
Yeah, the change in https://github.com/metabrainz/musicbrainz-server/pull/1252 works for me. I'm really confused why the rule I added broke - I tested it on several files and it was working when I added it. :\
